hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ClusterStateEndpoint.java

```diff
@@ -146,14 +146,14 @@ public Response getClusterState() {
       totalKeys += fileRecord.getValue();
     }
     if (deletedKeyRecord != null) {
-      deletedKeys += deletedKeyRecord.getValue();
+      keysAwaitingDeletion += deletedKeyRecord.getValue();
     }
     if (deletedDirRecord != null) {
       deletedDirs += deletedDirRecord.getValue();
     }
 
     builder.setKeys(totalKeys);
-    builder.setDeletedKeys(deletedKeys);
+    builder.setkeysAwaitingDeletion(keysAwaitingDeletion);
```

Comment:
```suggestion
    builder.setKeysAwaitingDeletion(keysAwaitingDeletion);
```

Reply:
@smengcl Completed Changes suggested in Review